physical_properties/README.md

```diff
@@ -1,15 +1,19 @@
-# Physical Properties
+# SAMPL6 Physicochemical Propery Prediction Challenges
 
-## Manifest
-- `potential_molecules_for_logD_challenge.xlsx`: Table of potential molecules for logD challenge, that includes structures and SMILES strings.
-- `potential_molecules_for_logD_challenge.pdf`: : Table of potential molecules for logD challenge, that includes structures and SMILES strings.
+## pKa Challenge
 
-## Potential molecules for logD challenge
+This challenge consists of predicting microscopic and macroscopic pKas of 24 small organic molecules. These fragment-like small molecules are selected for their similarity to kinase inhibitors and for experimental tractability. Our aim is to evaluate how well current pKa prediction methods perform with drug fragment-like molecules through blind predictions.
 
-SAMPL6 Physical Property prediction challenge will consist of predicting distribution coefficients of kinase inhibitor fragment-like small molecules.
-Currently Mehtap Isik from Dr. John Chodera's group is measuring experimental pKa and logD, in collaboration with Merck's Preformulation Group at Rahway.
+For detailed instructions for pKa challenge: [SAMPL6/pKa_challenge_instructions.md](https://github.com/MobleyLab/SAMPL6/blob/pKa/pKa_challenge_instructions.md)
 
-14 compounds from the larger small molecule set, that are currently experimented on, are provided in `potential_molecules_for_logD_challenge.xlsx` as examples. These should be regarded as potential compounds for logD challenge until experiments are finalized. Future updates will be posted regarding final small molecule set for logD prediction challenge upon completion of pKa and logD experiments.   
+Challenge start date: Oct 24, 2017 
+Challenge submission due: Jan 10, 2018  
 
 
+## logD Challenge
 
```

Comment:
It might be good to have a statement here that the logD challenge has been delayed until early/mid 2018, just incase new people join the repository.

Reply:
Done.